ocaml/tests/test_tar_ext.ml

```diff
@@ -0,0 +1,134 @@
+(*
+ * Copyright (c) Cloud Software Group, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+open Helpers
+open Tar_ext
+
+let default_max_size = !Xapi_globs.bundle_file_max_size
+
+let ( // ) = Filename.concat
+
+let gen_test_file_script = "test_data/gen_non_cdn_test_file.sh"
+
+let test_file_dir = "/tmp/test_tar_ext"
```

Comment:
This will run into problems if multiple instances of this code is run in parallel and access this directory. Creating a unique directory is not trivial, though. For testing this is probably ok.